docs/content/en/docs/concepts/tracing-policy/k8s-filtering.md

```diff
@@ -59,7 +59,7 @@ First, let us start minikube, build and load images, and install Tetragon and OC
 
 ```shell
 minikube start --container-runtime=containerd
-./contrib/rthooks/minikube-containerd-install-hook.sh
+./contrib/tetragon-rthooks/minikube-containerd-install-hook.sh
```

Comment:
cc @kkourt, we should have checked the doc at rename as well!

If you can check for others `/contrib/rthooks` it would be amazing